pkg/executor/distsql.go

```diff
@@ -784,6 +784,14 @@ func (e *IndexLookUpExecutor) startIndexWorker(ctx context.Context, initBatchSiz
 			builder.FullTextInfo.TableID = e.table.Meta().ID
 			builder.FullTextInfo.IndexID = e.index.ID
 			builder.FullTextInfo.ExecutorID = e.idxPlans[0].ExplainID().String()
+
+			id := e.Table().Meta().ID
+			// Mock Range  [t_id, t_id +1) to scan the whole table.
```

Comment:
Why do we say it's a mocked one here?
Should it not be full range?

Reply:
If we want to filter by handle, I think this keyrange needs to be changed